server/graphql-schema.js

```diff
@@ -740,6 +740,30 @@ const graphqlSchema = gql`
         conflictingResults: [PopulatedData]!
     }
 
+    type Issue {
+        """
+        GitHub username of the issue creator.
+        """
+        author: String!
+        """
+        Link to the GitHub issue's first comment.
+        """
+        link: String!
+        """
+        Indicates the type of issue. 'changes-requested' or 'feedback'.
+        'feedback' is the default type.
+        """
+        type: String!
```

Comment:
Not sure I follow this. It sounds like you're saying we should be able to expose all the labels retrieved from the GitHub issue through the GraphQL query. We're currently making use of the other labels within the resolver but I'm not what the need would be to expose all of them. Only the `changes-requested`/`feedback` label seem to matter to the client based on the latest designs.

Could you perhaps expand on what you meant here?

Reply:
Yeah, I was suggesting having an array of labels which would match how it's presented in GitHub. Then the client code would go something like `if (issue.labels.includes('changes-requested'))`.

If you think that's a bit too indirect, we could keep a single field but I would probably suggest having the word "label" in there since that's where the field is originating in GitHub. So `feedbackLabel` or something like that. I'm wary of the word "type" just because it gets thrown around a lot and doesn't have a lot of meaning in itself.